api/models/rules.py

```diff
@@ -0,0 +1,28 @@
+"""Rule model."""
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from .base import Base
+from .year import Year
+
+
+class Rules(Base):
+    """Stores rules, terms and conditions for year."""
+
+    year = models.ForeignKey(
+        Year,
+        verbose_name=_("Year"),
+    )
+    text = models.TextField(
```

Comment:
@PetrDlouhy, I don't know why are we going back to this again and again. We shouldn't develop features nobody asked for. It's a waste of time.